jobs/build/custom/Jenkinsfile

```diff
@@ -209,6 +215,30 @@ node {
                 }
             }
 
+            stage ('Attach Images') {
+                if (rpms.toUpperCase() == "NONE" && 
+                    params.IMAGES == "" &&
+                    params.IMAGE_MODE == "update-dockerfile") {
```

Comment:
I don't think it is reasonable to to restrict the condition for attaching builds.
Suppose we need to rebuild an image with newer RPMs/sources to catch up a CVE fix, we need to rerun the `build/custom` job with `SIGNED` option checked. It would be much convenient if this job can attach the build after job completes.